doc/admin-guide/files/ssl_server_name.yaml.en.rst

```diff
@@ -71,9 +71,18 @@ verify_client             One of the values :code:`NONE`, :code:`MODERATE`, or :
                           fail the TLS handshake if new certificate is presented.  If ``STRICT`` is specified
                           the client must resent a certificate during the TLS handshake.
 
-
                           By default this is :ts:cv:`proxy.config.ssl.client.certification_level`.
 
+valid_tls_protocols_in    This specifies the list of TLS protocols that will be offered to user agents during 
```

Comment:
This is quite a mouthful, would e.g. "tls_proto_in" not suffice?

Reply:
because the 1 syllable "cols" makes it expensive? It's clearer if the whole word is there (I'd guess especially to non-english as first language folks).

Reply:
This is only for TLS versions/protocols.  I'm ok with valid_tls_versions_in.